test/sparrow_test.exs

```diff
@@ -151,6 +152,7 @@ defmodule SparrowTest do
   end
 
   test "Sparrow starts correctly, FCM only", context do
+    {:ok, pid} = start_supervised(Sparrow.PoolsWarden)
```

Comment:
This looks lik a context for me. Can it be extracted there? See: https://hexdocs.pm/ex_unit/ExUnit.Case.html#module-context

Reply:
`pid` -> `_pid` as you do not use the variable ;)
```elixir
iex(1)> _pid = self()
#PID<0.103.0>
iex(2)> pid()
** (CompileError) iex:2: undefined function pid/0

iex(2)> pid = self()
#PID<0.103.0>
iex(3)> pid
#PID<0.103.0>
```